db/src/main/java/io/lantern/db/DB.kt

```diff
@@ -993,32 +1040,14 @@ class Transaction internal constructor(
     /**
      * Deletes the value at the given path
      */
-    fun <T : Any> delete(path: String, extractFullText: ((T) -> String)? = null) {
```

Comment:
Another nice benefit of this change is that we don't need `extractFullText` in order to be able to delete from the index.